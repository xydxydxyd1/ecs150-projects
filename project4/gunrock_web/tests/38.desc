Run ds3bits on a simple image
