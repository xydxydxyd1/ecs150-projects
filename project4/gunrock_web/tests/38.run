./ds3bits tests/disk_images/c.img
