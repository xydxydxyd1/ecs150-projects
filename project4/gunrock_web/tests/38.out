Super
inode_region_addr 3
inode_region_len 8
num_inodes 256
data_region_addr 11
data_region_len 32
num_data 32

Inode bitmap
255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 255 1 0 0 0 0 0 0 

Data bitmap
3 0 0 0 
